nixos/modules/services/games/minecraft-server.nix

```diff
@@ -158,6 +158,34 @@ in {
           + "-XX:MinHeapFreeRatio=5 -XX:MaxHeapFreeRatio=10";
         description = "JVM options for the Minecraft server.";
       };
+
+      standardInputFifo = mkOption {
+        type = types.str;
+        default = "/run/mincraft/stdin";
+        description = ''
+          The path to a FIFO for sending commands to standard input of the
+          Minecraft server process. E.g. this can be used to op players with
+          <command>echo "op Steve" > ''${standardInputFifo}</command>.
```

Comment:
Let's also add `"Note that only users in the <literal>minecraft</literal> group can run such commands."`